js/tests/unit/tooltip.js

```diff
@@ -680,6 +680,42 @@ $(function () {
     $styles.remove()
   })
 
+  test('should not be adjust position when scrolling', function () {
```

Comment:
But speaking generally, scrolling can affect tooltip position. E.g. An "auto top" tooltip will be displayed with position bottom if its associated element has been scrolled such that a top tooltip would get clipped by the top edge of the viewport.